konduit-serving-pipeline/src/test/java/ai/konduit/serving/pipeline/impl/data/DataTest.java

```diff
@@ -112,12 +125,117 @@ public void testEmbeddedData() {
         Data layeredContainer = JData.singleton("upperLevel", ndContainer);
     }
 
-    @Ignore
     @Test
     public void testSerde() {
         Data someData = JData.singleton(KEY, Long.valueOf(200));
-        someData.save(new File("temp"));
-        Data restoredData = JData.fromFile(new File("temp"));
+        try {
+            someData.save(new File("temp"));
+        } catch (IOException e) {
+            log.error("IOException in DataTest.testSerde()", e);
```

Comment:
2 problems here
(a) Don't use `new File("...")` in tests - this is something I flagged in the previous PR here: https://github.com/KonduitAI/konduit-serving/pull/297#discussion_r417772653
(b) Don't simply log exceptions, this is very bad... an exception can be thrown and the test can still pass.
In fact, we don't need try try/catch blocks at all - just use `public void testSerde() throws Exception` and let any exceptions propagate out and fail the test if they occur

Reply:
Fixed.